frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/stateUpdate/vmSettings/vm-settings-tab-state-update.ts

```diff
@@ -45,20 +44,11 @@ export const selectUserTemplateOnLoadedUpdater = ({
     return;
```

Comment:
you can remove the whole `selectUserTemplateOnLoadedUpdater` function

Reply:
\+ a bit of cleanup
- remove `initialized` from `VMSettingsField.USER_TEMPLATE` in `vm-settings-tab-initial-state.tsx`
- remove the `initialized` check from `selectedUserTemplateUpdater`

Reply:
done

Reply:
In addition, the form fields are reset when `None` is selected, so the form does not container values from former user template anymore.